### PR TITLE
docs for enterprise features det match and run incr

### PR DESCRIPTION
**Pull request by vikasgupta78**
docs for enterprise features det match and run incr

### CODE REVIEW

File: docs/deterministicMatching.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Zingg enterprise offers a new feature of deterministic matching along side already available probabilistic matching. In the entity resolution space, we refer to exact matching on attributes as deterministic matching. When used together this allows cases where user has given some sort of unique id in some rows to match accurately together due to deterministic matching and they will than match with others rows where id is missing via probabilistic matching.
+
+Example for configuring it in json:
+
+"deterministicMatching":[
+	{
+		"matchCondition":[
+			{
+				"fieldName":"fname"
```
Comment (sonalgoyal) [inline docs/deterministicMatching.md:11]: give an example containing two matchConditions

File: docs/deterministicMatching.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Deterministic Matching
+
+Zingg enterprise offers a new feature of deterministic matching along side already available probabilistic matching. In the entity resolution space, we refer to exact matching on attributes as deterministic matching. When used together this allows cases where user has given some sort of unique id in some rows to match accurately together due to deterministic matching and they will than match with others rows where id is missing via probabilistic matching.
```
Comment (sonalgoyal) [inline docs/deterministicMatching.md:3]: E caps

File: docs/deterministicMatching.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+				"fieldName":"add1"
+			}
+		]
+	}
+],
+
+
+Python code example:
+
+dm1 = DeterministicMatching('fname','stNo','add1')
```
[the remark this answers is not in the document]
Comment (sonalgoyal) [reply docs/deterministicMatching.md:26]: better naming of conditions I mean

File: docs/runIncremental.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	"config" : "<parent config used for match>",
+	"incrementalData": <data pipe for incremental data>
+}
+
+runIncremental can also be triggerred using python by invoking\
+`./scripts/zingg.sh --run examples/FebrlExample.py`
+
+Python code example:
+
+incrArgs = IncrementalArguments()
```
Comment (sonalgoyal) [inline docs/runIncremental.md:20]: this needs a complete example

File: docs/runIncremental.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+Zingg enterprise offers a new feature of running incremental load on already matched dataset. As new data arrives there is no need to run the match phase on entire data. Instead we can reuse the models created earlier on newer and incremental datasets. This will assign clusters on new data sets as well as update existing dataset and reassign clusters if needed.
+
+You can invoke the incremental phase by invoking\
+`./scripts/zingg.sh --phase runIncremental --conf <location to incrementalConf.json>`
+
+Example incrementalConf.json:
+
+{	
+	"config" : "<parent config used for match>",
```
Comment (sonalgoyal) [inline docs/runIncremental.md:11]: this has to be expanded with all the values

File: docs/deterministicMatching.md (hunk 1 above)
Comment (vikasgupta78) [reply docs/deterministicMatching.md]: done in commit 9a1a3f9f

File: docs/deterministicMatching.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (vikasgupta78) [reply docs/deterministicMatching.md]: done in commit 9a1a3f9f

File: docs/runIncremental.md (hunk 5 above)
Comment (vikasgupta78) [reply docs/runIncremental.md]: done in commit 9d27399e

File: docs/runIncremental.md (hunk 4 above)
Comment (vikasgupta78) [reply docs/runIncremental.md:74]: done in commit 9d27399e

Comment (sonalgoyal) [reply docs/runIncremental.md:74]: the example needs to be broken down into trainMatch and then running incremental using the same args. We also need comments in the code to highlight that. So it should show the flow

**Review by sonalgoyal — CHANGES_REQUESTED** [review]

**Comment by vikasgupta78** [comment]
for run incremental example would it be better to provide reference to https://github.com/zinggAI/zinggEnterprise/blob/main/spark/examples/databricks/FebrlExampleEnterprise.ipynb

We can copy this to OSS

**Pull request merged**